platform/osx/export/export.cpp

```diff
@@ -114,7 +114,7 @@ class EditorExportPlatformOSX : public EditorExportPlatform {
 	virtual void get_platform_features(List<String> *r_features) override {
 		r_features->push_back("pc");
 		r_features->push_back("s3tc");
-		r_features->push_back("OSX");
+		r_features->push_back("macOS");
```

Comment:
This line should be skipped on cherry-picking to 3.2.

Reply:
Thanks for the reminder!